Form/Type/SeoMetadataType.php

```diff
@@ -83,4 +95,28 @@ public function getName()
     {
         return 'seo_metadata';
     }
+
+    /**
+     * @param SeoMetadata $data
+     * @return null|SeoMetadataPersistenceModel
+     */
+    private function convertToPersisterModel(SeoMetadata $data)
+    {
+        if (!class_exists('Doctrine\Bundle\PHPCRBundle\DependencyInjection\Compiler\DoctrinePhpcrMappingsPass')
```

Comment:
No we don't really use it, we use the if-statemant for the existence of
the Doctrine classes and then use the parameter. But our Extension class
only set `backend_type_orm` OR `backend_type_phpcr`. So one of them
isn't available. I tried to set the oposite (backend_type_orm = false,
for phpcr), but then the the application was broken again.